content/kubermatic/master/guides/cluster_templates/_index.en.md

```diff
@@ -0,0 +1,32 @@
++++
+title = "Cluster templates"
+date = 2021-08-02T14:07:15+02:00
+weight = 20
+
++++
+
+## Understanding templates
+Cluster templates are designed to standardize and simplify the creation of Kubernetes clusters. A cluster template is a
+reusable cluster template object. It guarantees that every cluster it provisions from the template is uniform and consistent
+in the way it is produced.
+
+A cluster template allows you to specify a provider, node layout, and configurations to materialize a cluster instance
+via Kubermatic API or UI.
+
+## Scope
+The cluster templates are accessible from different levels.
+ - global: (mange by admin user) visible for everyone
+ - project: accessible for the project users
+ - user: accessible for the template owner
+
+Template management is available from project level.
+
+## Credentials
+Creating cluster from the template requires credentials to authenticate with the cloud provider. During template creation
+the credentials are stored in the secret which is assigned to the cluster template.
+
```

Comment:
Are they independent from other credentials? How does someone keep the overview?

Reply:
done